include/tscore/Allocator.h

```diff
@@ -97,16 +97,18 @@ class FreelistAllocator
     @param chunk_size number of units to be allocated if free pool is empty.
     @param alignment of objects must be a power of 2.
   */
-  FreelistAllocator(const char *name, unsigned int element_size, unsigned int chunk_size = 128, unsigned int alignment = 8)
+  FreelistAllocator(const char *name, unsigned int element_size, unsigned int chunk_size = 128, unsigned int alignment = 8,
+                    unsigned int use_hugepages = 0)
```

Comment:
Looks like I'm not getting my bools.

Reply:
What is the bool request again?

Reply:
To use bool instead of an integral type when the only values in the variable are 1 or 0.

Reply:
oh, ok.  Gotcha.